### PR TITLE
Add reference to REST API docs

### PR DESCRIPTION
**Pull request by bmansheim**
You can see the staging site at: https://docs.redislabs.com/staging/add-ref-to-rest-api-docs/rs/administering/

### CODE REVIEW

**Comment by bmansheim** [comment]
@Paz-Yanover Please review and comment.
https://docs.redislabs.com/staging/add-ref-to-rest-api-docs/rs/administering/

File: content/rs/administering/_index.md [hunk 1]
```diff
@@ -5,10 +5,24 @@ weight: 50
...
 ---
-This section covers everything you need to know to get up and running
-with RS, included installing, configuring a cluster, maintaining a
-cluster, creating databases, etc.
+This section covers everything you need to know to get up and running with RS.
 
-Topics include:
+{{% note %}}
+In addition to using the RS web UI to manage the cluster,
+you can also access cluster operations through the cluster REST API.
```
Comment (Paz-Yanover) [inline content/rs/administering/_index.md]: Access? is that the correct word here? not- perform?

File: content/rs/administering/_index.md [hunk 2]
```diff
@@ -5,10 +5,24 @@ weight: 50
...
+
+To access the cluster REST API documentation, either:
+
+- Open the HTML files in the RS software package:
+    1. Download the RS software package.
+    1. Unarchive the package.
```
Comment (Paz-Yanover) [inline content/rs/administering/_index.md]: I think we need to write here- install the RS.... (instead of the second point)

Comment (Paz-Yanover) [reply content/rs/administering/_index.md]: I noticed now that you mention it later. Is the first "option" approved by the Cluster team and working?

File: content/rs/administering/_index.md [hunk 3]
```diff
@@ -5,10 +5,24 @@ weight: 50
...
+    1. Unarchive the rlec_docs.tar.gz package.
+    1. Open the `rlec_docs\html\index.html` file in a web browser.
+- Open the HTML files in an installed RS node:
+    1. Connect to the terminal of an RS node.
+    1. Change directory to `/opt`.
```
Comment (Paz-Yanover) [inline content/rs/administering/_index.md]: Please check with Gideon whether this path is relevant for all supported OS

File: content/rs/administering/_index.md [hunk 4]
```diff
@@ -5,10 +5,24 @@ weight: 50
...
+    1. Download and unarchive the rlec_rest_api.tar.gz package.
```
Comment (Paz-Yanover) [inline content/rs/administering/_index.md]: Download? is should already be there

File: content/rs/administering/_index.md (hunk 1 above)
Comment (bmansheim) [reply content/rs/administering/_index.md]: @Paz-Yanover My intention was that you can get to the operations through the REST API, but I changed it to be more direct.

File: content/rs/administering/_index.md (hunk 3 above)
Comment (bmansheim) [reply content/rs/administering/_index.md]: @gidon@redislabs.com Do these instructions look OK to you?

File: content/rs/administering/_index.md (hunk 4 above)
Comment (bmansheim) [reply content/rs/administering/_index.md]: I meant to move it to another machine to view it in a browser. I made it more clear.

File: content/rs/administering/_index.md (hunk 3 above)
Comment (gidonmiller) [reply content/rs/administering/_index.md]: @bmansheim once the installer improvements are pushed the /opt path could be configured to another path. I assume there are more mentions of explicit paths in the documentation? if so we need to consider how to handle them in general.

Comment (bmansheim) [reply content/rs/administering/_index.md]: @gidon@redislabs.com There are many references to explicit paths in the docs. I'm going to merge this one. Please open a ticket in the context of the installer improvements about updating current docs to remove explicit paths.
cc: @banker

**Pull request merged**